module/designer/src/Domain/ValueObject/TemplateElement/AttributeTemplateElementProperty.php

```diff
@@ -35,8 +34,6 @@ public function __construct(AttributeId $attributeId, bool $required = false)
     }
 
     /**
-     * {@inheritDoc}
```

Comment:
👎🏻

Reply:
Why? We should interit docs from parent without PHP doc?